arangod/Cache/TransactionalBucket.cpp

```diff
@@ -198,7 +198,10 @@ void TransactionalBucket::evict(CachedValue* value, bool optimizeForInsertion) {
 
 void TransactionalBucket::clear() {
   TRI_ASSERT(isLocked());
-  memset(this, 0, sizeof(TransactionalBucket));
+  _state.clear();
+  memset(this + sizeof(BucketState), 0,
+         sizeof(TransactionalBucket) - sizeof(BucketState));
```

Comment:
Same problem as above - this would result in a write outside the bounds of the bucket instance.